llvm/test/CodeGen/AArch64/hwasan-check-memaccess-fixedshadow.ll

```diff
@@ -0,0 +1,144 @@
+; RUN: llc --hwasan-mapping-offset=4398046511104 < %s | FileCheck %s
+
+target triple = "aarch64--linux-android"
+
+define ptr @f1(ptr %x0, ptr %x1) {
+  ; CHECK: f1:
+  ; CHECK: str x30, [sp, #-16]!
+  ; CHECK-NEXT: .cfi_def_cfa_offset 16
+  ; CHECK-NEXT: .cfi_offset w30, -16
+  ; CHECK-NEXT: bl __hwasan_check_x1_1
```

Comment:
It looks autogenerated with llvm/utils/update_llc_test_checks.py
why there is no automatic comment in the top?

Reply:
I manually modified hwasan-check-memaccess.ll (per the commit message) and reviewed the changes for reasonableness. I did not use update_llc_test_checks.

Reply:
Would you like to try?

Example:
```
ninja -C <builddir> opt llc && llvm/utils/update_llc_test_checks.py --llc-binary <builddir>/bin/llc llvm-project/llvm/test/Transforms/RemoveTraps/remove-traps.ll
```

Reply:
Thanks, that's very helpful! Done.